src/themes/website/styles/navs.scss

```diff
@@ -75,6 +75,10 @@
     .nav-item {
         flex: 100%;
     }
+
+    &.margin-top {
```

Comment:
`.margin-top` could you please move this class to `utils.scss` and maybe rename it by analogy with similar classes. The problem is that we (all of us working on the UI) keep introducing same or similar classes and scatter them across the style files. Plus that meaning of those class names doesn't usually follow any pattern. Having them in one place at least gives us opportunity to find those patterns. Best of all, if we all learn about BEM model, which addresses exactly this problem. That would minimize usage of these utility classes.